client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/InteractedContactAccordion.tsx

```diff
@@ -35,20 +35,28 @@ const InteractedContactAccordion = (props: Props) => {
 
     const formErrors = errors?.form && errors?.form[index];
 
+    const generateBackgroundColorClass = (colorCode: Number | any) => {
+        switch (colorCode) {
+            case '1':
+                return classes.red;
+            case '2':
+                return classes.orange;
+            case '3':
+                return classes.green;
+            case '4':
+                return classes.yellow;
+            default:
+                return classes.white;
+        }
+    }
+
     const getAccordionClasses = (): string => {
         let classesList: string[] = [];
         classesList.push(classes.accordion);
 
-        const formHasErrors = formErrors
```

Comment:
why did you delete this section?
it is the color of the accordion when there are errors - not the ruler

Reply:
👍